src/dbcp/data_mart/projects.py

```diff
@@ -477,7 +490,7 @@ def _add_derived_columns(mart: pd.DataFrame) -> None:
         "Solar": "renewable",
         "Steam": np.nan,
         "Transmission": "transmission",
-        "Unknown": np.nan,
+        "Unknown": "other",
```

Comment:
Made this map to "other" because there are a decent number of withdrawn and operational projects that have "Unkown" fuel type. If they are mapped to null we lose a lot of the projects in the change log.